org.gpodder.gpodder-adaptive.json

```diff
@@ -0,0 +1,52 @@
+{
+  "app-id": "org.gpodder.gpodder-adaptive",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "42",
+  "sdk": "org.gnome.Sdk",
+  "command": "gpodder",
+  "rename-icon": "gpodder-adaptive",
+  "copy-icon": true,
+  "rename-desktop-file": "gpodder.desktop",
+  "finish-args": [
+    "--device=dri",
+    "--socket=x11",
+    "--socket=wayland",
+    "--share=ipc",
+    "--share=network",
+    "--own-name=org.gpodder.*",
+    "--talk-name=org.gtk.vfs.*",
+    "--filesystem=home"
+  ],
+  "cleanup": [
+    "/include",
+    "/lib/debug",
+    "/lib/pkgconfig"
+  ],
+  "modules": [
+    "python3-requirements.json",
+    "shared-modules/libappindicator/libappindicator-gtk3-introspection-12.10.json",
+    {
+      "name": "gpodder-adaptive",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/gpodder/gpodder/archive/refs/tags/adaptive/3.11.0+1.tar.gz",
+          "sha256": "f30aa3a477ffecf7cfc4b65913902556499d3710127cd10d12ff46a70321320b"
+        },
+        {
+          "type": "file",
+          "path": "appdata.xml"
```

Comment:
at has an appdata upstream. 

 https://github.com/gpodder/gpodder/blob/master/share/metainfo/org.gpodder.gpodder.appdata.xml

you shoudln't have one here

Reply:
True, the upstream appdata is outdated though, so fixing that requires a new release.

Reply:
or a patch.